wg-charter.html

```diff
@@ -0,0 +1,406 @@
+
+        <p>Additional technical coordination with the following Groups will be made, per the <a href="https://www.w3.org/2018/Process-20180201/#WGCharter">W3C Process Document</a>:</p>
+
+        <div>
+          <h3 id="w3c-coordination">W3C Groups</h3>
+          <dl>
+            <dt><a href="https://www.w3.org/2001/tag/">Technical Architecture Group (TAG)</a></dt>
+        <dd>The WG may ask the Technical Architecture Group to review some of its specifications.</dd>
+
+        <dt><a href="https://www.w3.org/2019/webapps/">Web Applications Working Group</a></dt>
+        <dd>The Web Applications Working Group (WebApps WG) produces specifications that facilitate the development of client-side web applications.</dd>
+
+        <dt><a href="https://www.w3.org/2011/webappsec/">Web Application Security Working Group</a></dt>
+        <dd>This Working Group develops security and policy mechanisms to improve the security of Web Applications, and enable secure cross-site communication.</dd>
+
+        <dt><a href="https://www.w3.org/wasm/">Web Assembly Working Group</a></dt>
+        <dd>This Working Group develops a size- and load-time-efficient format and execution environment, allowing compilation to the web with consistent behavior across a variety of implementations.</dd>
+
+        <dt><a href="https://www.w3.org/community/webmachinelearning/">Machine Learning for the Web Community Group</a></dt>
+        <dd>This Community Group incubates a dedicated low-level Web API for machine learning inference.</dd>
+
+        <dt><a href="https://www.w3.org/community/wicg/">Web Platform Incubator Community Group</a></dt>
+        <dd>
+          This group provides a lightweight venue for proposing, incubating and discussing new web platform features. The Web Performance Working group will incubate and review new proposals that are within scope of our charter within the WICG. Once such WICG-incubated proposal is implemented and available in at least one major browser, and has support from one more, it may be adopted by the Web Performance Working group.
+        </dd>
+          </dl>
+          <h3 id="external-coordination">External Organizations</h3>
+          <dl>
+            <dt><a href="https://www.ecma-international.org/memento/tc39.htm" id="tc39">ECMA Technical Committee 39 (TC39)</a></dt>
```

Comment:
Another residual case of "Web Performance Working Group" here. Also, does the group really expect to work closely with ECMA? Are there going to be new language capabilities brought through this WG? If this is a typical listed liason group, that's fine. However, it'd be more likely that this group would need to work with the folks building the WebIDL specification than folks at ECMA directly.

Reply:
Same here. I dropped that in latest PR.

Reply:
:)